build.xml

```diff
@@ -92,8 +92,8 @@ Authors:
     <property name="optimize" value="on"/>
     <property name="deprecation" value="off"/>
     
-    <property name="javac.source" value="1.8"/>
-    <property name="javac.target" value="1.8"/>
+    <property name="javac.source" value="1.7"/>
```

Comment:
see https://github.com/apache/xerces-j/pull/60

let's keep different issues in different PRs